e2e/playwright/fixtures/toolbarFixture.ts

```diff
@@ -212,8 +212,8 @@ export class ToolbarFixture {
   }
   selectArc = async () => {
     await this.page.getByRole('button', { name: 'caret down arcs:' }).click()
-    await expect(this.page.getByTestId('dropdown-arc')).toBeVisible()
-    await this.page.getByTestId('dropdown-arc').click()
+    await expect(this.page.getByTestId('dropdown-tangential-arc')).toBeVisible()
+    await this.page.getByTestId('dropdown-tangential-arc').click()
```

Comment:
The old test ID was deleted (those are supposed to be permanent 😅 you know...) but I'm not positive this is the correct new value.

Reply:
I guess it's correct because the test is now passing: https://test-analysis-bot.hawk-dinosaur.ts.net/projects/KittyCAD/modeling-app/tests/170?branch=draft-snapshot-fixes&expand=false

Reply:
uhhh I don't even see this `id` unless it is generated from a runtime string :(

<img width="939" height="170" alt="Image" src="https://github.com/user-attachments/assets/355da24b-1cf8-4696-9816-c098f2c3920b" />

Reply:
Those test IDs are generated dynamically: https://github.com/KittyCAD/modeling-app/blob/main/src/lib/toolbar.ts